src/Components/Button/styles.ts

```diff
@@ -14,13 +14,34 @@ export const ButtonStyle = styled.button`
   transition-property: background-color;
   transition-duration: 300ms;
   &:hover {
-    background-color: #ffdba2;
+    background-color: #facdd4;
     transition-duration: 300ms;
+    color:#ef7c8e;
   }
 `;
 export const ButtonClear = styled(ButtonStyle)`
-  margin-top: 15px;
-  margin-left: 80%;
+  margin-left: 11px;
```

Comment:
[fix] It's not a good practice to use `px` for margins, font sizes, paddings and so on as it makes them static, this way if the user configures their browser to use a bigger font your application won't be affected. You can use `rem` or `em`. You can install the `sainoba.px-to-rem` extension in vscode and let it convert the `px` values to `rem` automatically.

Reply:
Thanks! Just updated all of them.